modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -1,73 +1,111 @@
 [[contact-methods-saltssh]]
-= Push via Salt SSH
+= SSH Push contact method
 
 
-Push via Salt SSH is used in environments where Salt clients cannot reach the {productname} Server directly.
+SSH Push ([literal]``ssh-push``) is used in environments where Salt clients cannot reach the {productname} Server directly.
 In this environment, clients are located in a firewall-protected zone called a DMZ.
 No system within the DMZ is authorized to open a connection to the internal network  where the {productname} Server is located.
 
-Push via Salt SSH is also to use if installing a daemon agent on clients is not possible.
+SSH Push is also to use if installing a daemon agent on clients is not possible.
 
 [WARNING]
 ====
-The Push via Salt SSH method has serious limitations.
-It does not scale well, and consumes more Server resources and network bandwidth than the plain Salt method.
+The SSH Push method has serious limitations.
+It does not scale well, and consumes more Server resources and network bandwidth than the plain Salt method ([literal]``default``).
 The Push via Salt SSH method is not at all supported with large setups (1000 clients and more).
 ====
 
-The Push via Salt SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located in the DMZ.
-After all actions such as updates and events are pushed and executed, the tunnel is closed.
+The server uses the SSH Push to contact the clients at regular intervals, checking in and performing scheduled actions and events.
 
-The server uses the Salt SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
-
-////
-This contact method works for Salt clients only.
-For traditional clients, use Push via SSH.
-////
-
-This image demonstrates the Push via Salt SSH process path.
+This image demonstrates the Push via SSH Push process path.
 All items left of the [systemitem]``Taskomatic`` block represent processes running on the {productname} client.
 
 image::salt-ssh-contact-taigon.png[scaledwidth=80%]
```

Comment:
Is this image loading? I see the broken link in the GitHub file preview, but I'm not sure.

Reply:
Yes, it loads if you use the output generated with our antora tool chain.  I think github isn't completely aware of directory structure; thus displaying images or `{macro}` expansion often doesn't work with the built-in viewer.
Thanks for feedback and your review, Yeray!